apps/meteor/tests/e2e/embedded-layout.spec.ts

```diff
@@ -4,78 +4,124 @@ import { createDirectMessage, createTargetChannel, deleteChannel } from './utils
 import { test, expect } from './utils/test';
 
 const embeddedLayoutURL = (pageUrl: string) => `${pageUrl}?layout=embedded`;
+
 test.use({ storageState: Users.user1.state });
 
-test.describe.serial('embedded-layout', () => {
+test.describe.parallel('embedded-layout', () => {
```

Comment:
```suggestion
test.describe('embedded-layout', () => {
```

https://playwright.dev/docs/api/class-test#test-describe-parallel